streamwebs_frontend/streamwebs/templates/streamwebs/datasheets/macroinvertebrate_view.html

```diff
@@ -0,0 +1,222 @@
+{% load staticfiles %}
+
+<!DOCTYPE html>
```

Comment:
We have a base template now that you can add. You might also want to add [``trans`` template tags](https://docs.djangoproject.com/en/1.9/topics/i18n/translation/#trans-template-tag) for all your hardcoded labels so that they're translatable in the future.